portfolio/src/main/webapp/style.css

```diff
@@ -86,6 +93,10 @@ h6 {
   width: 33.33%;
 }
 
+#myDIV {
```

Comment:
Is it intended to have 2 declarations of myDIV in this file?

Reply:
I did mean to do this.  This is the code I wrote for displaying the picture of my chicken and  I and since there were two states (one when you couldn't see it and one where the image was revealed)  I made two separate declarations. This seemed to work , however, I am not sure if there is a superior what to go about doing this.

Reply:
Interesting. From my understanding you shouldn't need this declaration, your javascript should be able to hide it correctly already. Could you let me know if that is not the case?

Reply:
Done